### DISCUSSION TITLE
Incorrect Flattening of Iceland and the Faroe Islands

### DISCUSSION THREAD

**Discussion by Andreweag** [kubilus1/autoortho#303] [General]
Here are some screenshots of the difference, curious if anyone else was experiencing this. I also remember some discussions about this on the org, but those were for issues with the altitude data w/ Ortho4XP, not AutoOrtho. Does anyone have a solution for an issue that could be on my end, or is it an issue with the source files (if everyone is experiencing this.) Also, As well as testing Iceland and the Faroe Islands, Northern Scotland was normal, as was Norway. However, I haven't checked north-east canada (hudson bay area) or greenland. I understand that there is a point where scenery stops generating, but as the norway scenery is there considerably far north (I've yet to see it go away,) I doubt that this is the issue I'm encountering, especially as default is there. 

![Screenshot 2023-08-01 205546](https://github.com/kubilus1/autoortho/assets/92647980/dde4f354-df8e-4263-ad32-5185552dfe1f)
![Screenshot 2023-08-01 205827](https://github.com/kubilus1/autoortho/assets/92647980/367466b3-629a-4f4f-9a1f-06175f8742cd)
Also note that in the image with autoortho, everything is completely at sea level, although the 'shading' on the terrain might make it appear otherwise.

**Comment by d41k4n** [comment]
AutoOrtho scenery is pre-built with Ortho4XP and therefore it will suffer from exactly the same issues and limitations unless certain settings are customized during the scenery authoring process or the user re-generates the problematic tiles with Ortho4XP using custom settings and without breaking compatibility to the existing AO install.

This particular issue has the same root cause and fix as already discussed in countless topics in the Ortho4XP support forum e.g.

https://forums.x-plane.org/index.php?/forums/topic/171142-iceland-mesh/
https://forums.x-plane.org/index.php?/forums/topic/185143-iceland-always-flat-hgt-files-missing/
https://forums.x-plane.org/index.php?/forums/topic/224033-the-faroe-islands-and-icelands-tiles-have-flat-terrain-literally/

etc.

**Comment by Andreweag** [comment]
Ok, I'll try creating my own tiles. Would there be any way to modify or add an 'overlay' to the source files of autoortho that'd fix the issue?

**Comment by d41k4n** [comment]
No, there's no way around recreating the base mesh DSFs I'm afraid.

**Reply by Andreweag** [reply]
Ok thanks anyways
